Dockerfile

```diff
@@ -1,31 +1,54 @@
-FROM public.ecr.aws/p9i6h6j0/aws-pdk:latest as build
+FROM --platform=linux/x86_64 public.ecr.aws/amazonlinux/amazonlinux:latest as build
 
-#install dependencies
-RUN dnf -y install rsync
+RUN dnf install tar gzip python3 gcc-c++ make python3-pip rsync shadow-utils -y
+
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 22
```

Comment:
Please use node version 20 to be consistent with Github actions (we are about to update it to node 20)